flask/migrations/versions/2b0a14520510_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
+
+Revision ID: 2b0a14520510
+Revises: 1e8a9b118c49
+Create Date: 2021-01-06 22:49:34.132758
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '2b0a14520510'
+down_revision = '1e8a9b118c49'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
can we delete these comments

Reply:
everything in the migrations folder is auto-generated. We should leave it as it is

Reply:
no we don't need those please adjust and end commands comments